catalog/gallery/lib/catalog/catalog_scaffold_screen.dart

```diff
@@ -17,22 +17,22 @@ class CatalogScaffold extends StatelessWidget {
   @override
   Widget build(BuildContext context) => Scaffold(
         appBar: AppBar(
-          backgroundColor: context.theme.colors.background,
+          backgroundColor: context.theme.colors.primary.shade400,
           leading: showBack ?? true
               ? IconButton(
                   icon: Icon(
                     Icons.arrow_back,
-                    color: context.theme.colors.primary,
+                    color: context.theme.colors.primary.shade100,
                   ),
                   onPressed: () => context.router.pop(),
                 )
               : null,
           title: Text(
             title,
-            style: TextStyle(color: context.theme.colors.primary),
+            style: TextStyle(color: context.theme.colors.textColor.shade100),
           ),
         ),
-        backgroundColor: context.theme.colors.background,
+        backgroundColor: context.theme.colors.primary.shade200,
         body: SafeArea(
           child: child,
```

Comment:
What about adding padding so the items don't look streched?
<img width=200 src="https://github.com/xmartlabs/flutter-template/assets/50491531/d1722f3a-2cd8-4cfa-9af4-8981deb4f8b6"/>

kind of like this: 
```suggestion
       Padding(
         padding: EdgeInsets.symmetric(horizontal: 16),
         child: child
```

Reply:
I agree, in the next pr i will change it !